Tests/AppTests/Common/Helpers/Controllers/Registration/RegFieldRuleHelper.swift

```diff
@@ -0,0 +1,38 @@
+import Vapor
+import Fluent
+
+@testable import App
+
+final class RegFieldRuleHelper {
+  
+  static func assertRegFieldRuleHasExpectedFields(_ rules: JSON) throws -> Bool {
+    guard
+      let _ = rules["id"]?.int,
+      let _ = rules["type"]?.string
+      else {
+        return false
+    }
+    return true
```

Comment:
this whole method can be rewrite to:
`return rules["id"]?.int != nil && rules["type"]?.string != nil`

Reply:
fixed, thanks